tests/component/test_running_command.py

```diff
@@ -18,7 +18,7 @@ def test_pipeline_runs_successfully(run_path):
     THEN the pipeline runs successfully
         And returns a return code of 0 within specified timeout
     """
-    time_limit_seconds = 9
```

Comment:
We need a `TODO` here to reduce this time in future PR

Reply:
done!